lib/audit_log/audit_log/formatter.rb

```diff
@@ -56,12 +56,18 @@ def sidekiq_current_ctx
               Sidekiq::Context.current
             else
               # versions up to 6.0.0
-              Thread.current[:sidekiq_context]
+              parse_sidekiq_ctx(Thread.current[:sidekiq_context])
             end
-          rescue NoMethodError
+          rescue StandardError
             nil
           end
 
+          def parse_sidekiq_ctx(ctx)
+            return unless ctx
+
+            ctx.last.match(/^(?<class>[^\s]+) JID-(?<jid>[^\s]+)/)
```

Comment:
Prefer a constant to a literal:

```suggestion
            ctx.last.match(SIDEKIQ_CONTEXT_RE)
```

Reply:
Good suggestion. This part was already done via #772